src/openrct2/peep/peep.c

```diff
@@ -9918,6 +9926,10 @@ static void get_ride_queue_end(sint16 *x, sint16 *y, sint16 *z){
 	rct_xy16 result = { 0, 0 };
 	rct_map_element *mapElement = map_get_first_element_at(*x / 32, *y / 32);
 
+	if (mapElement == NULL) {
+		return;
```

Comment:
What happens in the calling function if this returns without setting the x/y/z vars? I know its done later in the function as well but it would be good to just double check that this is fine.